### DISCUSSION TITLE
Finding mnemonics from private key or accessing wallet with private key, please HELP!

### DISCUSSION THREAD

**Discussion by Dev1LchE** [Chia-Network/chia-blockchain#8632] [Support]
Lets begin with me being a stupid idiot....

I was reinstalling windows on a machine that runs farming and plotting for my small farm. I know that I should have saved 24 words mnemonic and I thought I did, but apparently I've only copied all info that you get when you run command chia.exe keys show, 
hence I ended up with saving all my keys without getting my mnemonics 24 words seed. And ofcourse now I have a problem to access my wallet.
Considering that private key is made from encryption of mnemonic seed there should be a possibility to get your 24 words from private key itself ? Is that possible ? Someone please help.... I'm not even pissed for money I have on the account, I'm pissed because now I need to replot all plots because they are attached to this account...

**Comment by BrandtH22** [comment]
@Dev1LchE , unfortunately there is no way to go from the private key to the mnemonic. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
